samtranslator/swagger/swagger.py

```diff
@@ -28,6 +28,7 @@ class SwaggerEditor(object):
     _CACHE_KEY_PARAMETERS = "cacheKeyParameters"
     # https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html
     _ALL_HTTP_METHODS = ["OPTIONS", "GET", "HEAD", "POST", "PUT", "DELETE", "PATCH"]
+    _EXCLUDED_PATHS_FIELDS = ["summary", "description", "parameters"]
```

Comment:
Looking at https://swagger.io/specification/#path-item-object, should `servers` be in this list as well?  As well, should we add `trace` to `_ALL_HTTP_METHODS`?

Reply:
That's true.  The thinking was that SAM could support the full standard and then SAM doesn't need to make adjustments if API GW adds support for `trace` (or other parts of the OpenAPI spec.)  In any case, it's a minor point.